### DISCUSSION TITLE
Provide IT-Hub translations for Major Release

### DISCUSSION THREAD

**Discussion by JoCa96** [SchwarzIT/onyx#848] [Ideas]
Idea of @jannick-ux: We should provide all IT-Hub languages for the major release.

**Comment by larsrickert** [comment]
We could also ask in our web developer community if some people are capable of translating some languages.
Possible languages:
- Spanish
- Bulgarian
- French
- ...

**Comment by mj-hof** [comment]
Maybe a relevant question that can be answered by our community

**Comment by bergser** [comment]
Here is the list with locales we need to support in our app:
- en-US
- en-GB
- en-HK
- en-IE
- en-MT
- en-SG
- de-DE
- de-AT
- de-CH
- es-ES
- es-US
- fr-BE
- fr-CH
- fr-FR
- it-CH
- it-IT
- nl-BE
- nl-NL
- bg-BG
- bs-BA
- cs-CZ
- da-DK
- el-GR
- et-EE
- fi-FI
- hr-HR
- hu-HU
- lt-LT
- lv-LV
- mk-MK
- pl-PL
- pt-PT
- ro-RO
- sk-SK
- sl-SI
- sr-RS
- sv-SE

**Comment by larsrickert** [comment]
We will introduce new translation in #4141 👍  I will close this discussion and you can track the progress there
